patches/attendance/styles.css

```diff
@@ -0,0 +1,29 @@
+.attendance-tabs > button {
+    padding-inline: 15px;
+    padding-block: 10px;
+    font-size: 15px;
+    background: var(--kolor-wcag);
```

Comment:
![opera_gZAIWUxHP2](https://github.com/user-attachments/assets/a781c421-a36c-4e70-8fa5-510af345b4b0)
![opera_zRNMz6NcKQ](https://github.com/user-attachments/assets/3598cb53-23e5-4243-8186-7f10d98b617a)


Buttons doesn't look good with this background on eduVULCAN, maybe change color of font?

Reply:
imo it looks good enough

Reply:
ok